### PR TITLE
Hack to give preference to less loaded nodes

### PR DESCRIPTION
**Pull request by schmichael**

### CODE REVIEW

File: balancer.go [hunk 1]
```diff
@@ -111,6 +111,10 @@ func (e *FairBalancer) CanClaim(taskid string) (time.Time, bool) {
 		// Return delay set by Balance()
 		return e.delay, false
 	}
+
+	// Sleep proportional to number of tasks
+	n := len(e.bc.Tasks())
+	time.Sleep(time.Duration(n>>2) * time.Millisecond)
```
Comment (epsniff) [inline balancer.go]: hey I like the shift method, so we dn't get divide by zero errors

**Comment by epsniff** [comment]
:+1: just in case you go this route

**Comment by Ropes** [comment]
token :+1:

**Comment by schmichael** [comment]
Merging but made issue #125 to hopefully come up with a better solution later.

**Pull request merged**